### PR TITLE
Add `global_ultimate_duns_number` field to Company

### PR DESCRIPTION
**Pull request by alixedi**
### Description of change

This field will be used to store the `duns_number` for the global ultimate. We get this from D&B. Code for ingesting it in following PR.

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.md](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.md) for instructions
* [ ] Do any added or updated endpoints appear in the API documentation? See [docs/Maintaining the API documentation.md](https://github.com/uktrade/data-hub-leeloo/blob/develop/docs/Maintaining&#32;the&#32;API&#32;documentation.md) for more details
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/company/models/company.py [hunk 1]
```diff
@@ -239,6 +239,18 @@ class Company(ArchivableModel, BaseModel):
         choices=EXPORT_POTENTIAL_SCORES,
         help_text='Score that signifies export potential, imported from Data Science',
     )
+    global_ultimate_duns_number = models.CharField(
+        blank=True,
+        null=True,
+        help_text='Dun & Bradstreet unique identifier for global ultimate.',
+        max_length=9,
+        unique=True,
```
Comment (reupen) [inline datahub/company/models/company.py]: Is this meant to have `unique=True`? Doesn't that mean a global ultimate can only have one subsidiary?

Comment (alixedi) [reply datahub/company/models/company.py]: Good spot! Fixing now.

File: datahub/company/migrations/0095_company_global_ultimate_duns_number.py [hunk 2]
```diff
@@ -20,7 +20,6 @@ class Migration(migrations.Migration):
                 help_text='Dun & Bradstreet unique identifier for global ultimate.',
                 max_length=9,
                 null=True,
```
Comment (reupen) [inline datahub/company/migrations/0095_company_global_ultimate_duns_number.py]: FYI, we haven't generally been using `null=True` on `CharField`s (when `unique=False`) in line with this: https://docs.djangoproject.com/en/2.2/ref/models/fields/#null

File: datahub/company/models/company.py (hunk 1 above)
Comment (currycoder) [reply datahub/company/models/company.py]: Whoops - I missed this too. Might be good to have an index instead still..

Comment (reupen) [reply datahub/company/models/company.py]: Yep, good point, assume we're planning to filter by it so `db_index=True` would indeed be good.

**Pull request merged**